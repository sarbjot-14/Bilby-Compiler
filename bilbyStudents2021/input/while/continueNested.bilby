main{
	mut count := 0;
	mut count2 := 0;
	while (count < 3) {
		print "outer"$s count $n;
		count := count +1;
		while (count2 < 4) {
			
			count2 := count2 +1;
			if(count2 ==3){
				print "THREE" $n;
				continue;
			}
			print "inner" $s count2 $n;
		}
		count2 := 0;
		
		
	}
}