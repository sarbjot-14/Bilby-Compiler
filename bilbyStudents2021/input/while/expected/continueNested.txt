outer 0
inner 1
inner 2
THREE
inner 4
outer 1
inner 1
inner 2
THREE
inner 4
outer 2
inner 1
inner 2
THREE
inner 4
